schemas/20-cloud-extra.sql

```diff
@@ -53,17 +27,33 @@ PARTITION p2016_08 VALUES LESS THAN (TO_DAYS('2016-09-01')),
 PARTITION p2016_09 VALUES LESS THAN (TO_DAYS('2016-10-01')),
 PARTITION p2016_10 VALUES LESS THAN (TO_DAYS('2016-11-01')),
 PARTITION p2016_11 VALUES LESS THAN (TO_DAYS('2016-12-01')),
-PARTITION p2017_01 VALUES LESS THAN (TO_DAYS('2017-01-01')),
-PARTITION p2017_02 VALUES LESS THAN (TO_DAYS('2017-02-01')),
-PARTITION p2017_03 VALUES LESS THAN (TO_DAYS('2017-03-01')),
-PARTITION p2017_04 VALUES LESS THAN (TO_DAYS('2017-04-01')),
-PARTITION p2017_05 VALUES LESS THAN (TO_DAYS('2017-05-01')),
-PARTITION p2017_06 VALUES LESS THAN (TO_DAYS('2017-06-01')),
-PARTITION p2017_07 VALUES LESS THAN (TO_DAYS('2017-07-01')),
-PARTITION p2017_08 VALUES LESS THAN (TO_DAYS('2017-08-01')),
-PARTITION p2017_09 VALUES LESS THAN (TO_DAYS('2017-09-01')),
-PARTITION p2017_10 VALUES LESS THAN (TO_DAYS('2017-10-01')),
-PARTITION p2017_11 VALUES LESS THAN (TO_DAYS('2017-11-01')),
-PARTITION p2017_12 VALUES LESS THAN (TO_DAYS('2017-12-01')),
+PARTITION p2016_12 VALUES LESS THAN (TO_DAYS('2017-01-01')),
+PARTITION p2017_01 VALUES LESS THAN (TO_DAYS('2017-02-01')),
+PARTITION p2017_02 VALUES LESS THAN (TO_DAYS('2017-03-01')),
+PARTITION p2017_03 VALUES LESS THAN (TO_DAYS('2017-04-01')),
+PARTITION p2017_04 VALUES LESS THAN (TO_DAYS('2017-05-01')),
+PARTITION p2017_05 VALUES LESS THAN (TO_DAYS('2017-06-01')),
+PARTITION p2017_06 VALUES LESS THAN (TO_DAYS('2017-07-01')),
+PARTITION p2017_07 VALUES LESS THAN (TO_DAYS('2017-08-01')),
+PARTITION p2017_08 VALUES LESS THAN (TO_DAYS('2017-09-01')),
+PARTITION p2017_09 VALUES LESS THAN (TO_DAYS('2017-10-01')),
+PARTITION p2017_10 VALUES LESS THAN (TO_DAYS('2017-11-01')),
+PARTITION p2017_11 VALUES LESS THAN (TO_DAYS('2017-12-01')),
+PARTITION p2017_12 VALUES LESS THAN (TO_DAYS('2018-01-01')),
+PARTITION p2018_01 VALUES LESS THAN (TO_DAYS('2018-02-01')),
+PARTITION p2018_02 VALUES LESS THAN (TO_DAYS('2018-03-01')),
+PARTITION p2018_03 VALUES LESS THAN (TO_DAYS('2018-04-01')),
+PARTITION p2018_04 VALUES LESS THAN (TO_DAYS('2018-05-01')),
+PARTITION p2018_05 VALUES LESS THAN (TO_DAYS('2018-06-01')),
+PARTITION p2018_06 VALUES LESS THAN (TO_DAYS('2018-07-01')),
+PARTITION p2018_07 VALUES LESS THAN (TO_DAYS('2018-08-01')),
+PARTITION p2018_08 VALUES LESS THAN (TO_DAYS('2018-09-01')),
+PARTITION p2018_09 VALUES LESS THAN (TO_DAYS('2018-10-01')),
+PARTITION p2018_10 VALUES LESS THAN (TO_DAYS('2018-11-01')),
+PARTITION p2018_11 VALUES LESS THAN (TO_DAYS('2018-12-01')),
+PARTITION p2018_12 VALUES LESS THAN (TO_DAYS('2019-01-01')),
+PARTITION p2019_01 VALUES LESS THAN (TO_DAYS('2019-02-01')),
+PARTITION p2019_02 VALUES LESS THAN (TO_DAYS('2019-03-01')),
+PARTITION p2019_03 VALUES LESS THAN (TO_DAYS('2019-04-01')),
```

Comment:
We should try and automate this. It's a bit silly having to visually check these!

Reply:
+1